src/server/config/schema.js

```diff
@@ -30,9 +30,9 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
-  uuid: Joi.string().guid().default(),
 
   server: Joi.object({
+    uuid: Joi.string().guid().default(),
```

Comment:
Good point. The 5.0 alphas had `uuid` at the root of the config, so if someone had the uuid in their config, they'd get a validation error when they upgraded.

It's hard to say how many people would have added that config. It isn't actually documented anywhere.